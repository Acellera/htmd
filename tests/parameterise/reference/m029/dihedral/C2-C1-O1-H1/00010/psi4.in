set {
	reference rhf
	basis 6-31G*
}


set_num_threads( 1 )
memory 2 gb

molecule MOL {
0 1

H	 2.652209	 0.763231	 0.000000
O	 2.263154	 -0.100023	 0.000000
C	 0.912879	 -0.015685	 0.000000
C	 0.287924	 0.237824	 1.213979
C	 0.158440	 -0.171963	 -1.151296
H	 0.891354	 0.355742	 2.094761
C	 -1.089181	 0.332796	 1.264682
H	 0.643963	 -0.368596	 -2.092638
C	 -1.223823	 -0.074066	 -1.085859
H	 -1.568425	 0.529252	 2.207240
C	 -1.856472	 0.178032	 0.117412
H	 -1.802171	 -0.196668	 -1.984287
H	 -2.927393	 0.253270	 0.164666

	symmetry c1
}
ee = energy('scf')
oeprop('DIPOLE', 'QUADRUPOLE', 'MULLIKEN_CHARGES')
f=open( 'psi4out.xyz', 'w' )
f.write( "13  " )
f.write( str(ee) + "\n" )
f.write( MOL.save_string_xyz() )
f.close()
