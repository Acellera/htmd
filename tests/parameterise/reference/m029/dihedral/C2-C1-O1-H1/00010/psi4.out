    -----------------------------------------------------------------------
          Psi4: An Open-Source Ab Initio Electronic Structure Package
                              Psi4 (no tag) Driver

                          Git: Rev {detached?}  

    J. M. Turney, A. C. Simmonett, R. M. Parrish, E. G. Hohenstein,
    F. A. Evangelista, J. T. Fermann, B. J. Mintz, L. A. Burns, J. J. Wilke,
    M. L. Abrams, N. J. Russ, M. L. Leininger, C. L. Janssen, E. T. Seidl,
    W. D. Allen, H. F. Schaefer, R. A. King, E. F. Valeev, C. D. Sherrill,
    and T. D. Crawford, WIREs Comput. Mol. Sci. 2, 556-565 (2012)
    (doi: 10.1002/wcms.93)

                         Additional Contributions by
    A. E. DePrince, M. Saitow, U. Bozkaya, A. Yu. Sokolov
    -----------------------------------------------------------------------


    Psi4 started on: Tue Jul  5 16:26:29 2016

    Process ID:  12036
    PSI4DATADIR: /export131/home/mjharvey/miniconda3/share
    Memory level set to 256.000 MB

  ==> Input File <==

--------------------------------------------------------------------------
set {
	reference rhf
	basis 6-31G*
}


set_num_threads( 1 )
memory 2 gb

molecule MOL {
0 1

H	 2.652209	 0.763231	 0.000000
O	 2.263154	 -0.100023	 0.000000
C	 0.912879	 -0.015685	 0.000000
C	 0.287924	 0.237824	 1.213979
C	 0.158440	 -0.171963	 -1.151296
H	 0.891354	 0.355742	 2.094761
C	 -1.089181	 0.332796	 1.264682
H	 0.643963	 -0.368596	 -2.092638
C	 -1.223823	 -0.074066	 -1.085859
H	 -1.568425	 0.529252	 2.207240
C	 -1.856472	 0.178032	 0.117412
H	 -1.802171	 -0.196668	 -1.984287
H	 -2.927393	 0.253270	 0.164666

	symmetry c1
}
ee = energy('scf')
oeprop('DIPOLE', 'QUADRUPOLE', 'MULLIKEN_CHARGES')
f=open( 'psi4out.xyz', 'w' )
f.write( "13  " )
f.write( str(ee) + "\n" )
f.write( MOL.save_string_xyz() )
f.close()
--------------------------------------------------------------------------

  Memory set to   2.000 GiB by Python script.

*** tstart() called on cx1-3-2-1.cx1.hpc.ic.ac.uk
*** at Tue Jul  5 16:26:29 2016


         ---------------------------------------------------------
                                   SCF
            by Justin Turney, Rob Parrish, and Andy Simmonett
                              RHF Reference
                        1 Threads,   2000 MiB Core
         ---------------------------------------------------------

  ==> Geometry <==

    Molecular point group: c1
    Full point group: C1

    Geometry (in Angstrom), charge = 0, multiplicity = 1:

       Center              X                  Y                   Z               Mass       
    ------------   -----------------  -----------------  -----------------  -----------------
           H          2.648495956230     0.703788498337    -0.049975697026     1.007825032070
           O          2.259440956230    -0.159465501663    -0.049975697026    15.994914619560
           C          0.909165956230    -0.075127501663    -0.049975697026    12.000000000000
           C          0.284210956230     0.178381498337     1.164003302974    12.000000000000
           C          0.154726956230    -0.231405501663    -1.201271697026    12.000000000000
           H          0.887640956230     0.296299498337     2.044785302974     1.007825032070
           C         -1.092894043770     0.273353498337     1.214706302974    12.000000000000
           H          0.640249956230    -0.428038501663    -2.142613697026     1.007825032070
           C         -1.227536043770    -0.133508501663    -1.135834697026    12.000000000000
           H         -1.572138043770     0.469809498337     2.157264302974     1.007825032070
           C         -1.860185043770     0.118589498337     0.067436302974    12.000000000000
           H         -1.805884043770    -0.256110501663    -2.034262697026     1.007825032070
           H         -2.931106043770     0.193827498337     0.114690302974     1.007825032070

  Running in c1 symmetry.

  Rotational constants: A =      0.19168  B =      0.08838  C =      0.06081 [cm^-1]
  Rotational constants: A =   5746.28459  B =   2649.48383  C =   1822.93964 [MHz]
  Nuclear repulsion =  273.101926143630180

  Charge       = 0
  Multiplicity = 1
  Electrons    = 50
  Nalpha       = 25
  Nbeta        = 25

  ==> Algorithm <==

  SCF Algorithm Type is DF.
  DIIS enabled.
  MOM disabled.
  Fractional occupation disabled.
  Guess Type is CORE.
  Energy threshold   = 1.00e-06
  Density threshold  = 1.00e-06
  Integral threshold = 0.00e+00

  ==> Primary Basis <==

  Basis Set: 6-31G*
    Number of shells: 54
    Number of basis function: 117
    Number of Cartesian functions: 117
    Spherical Harmonics?: false
    Max angular momentum: 2

  ==> Pre-Iterations <==

   -------------------------------------------------------
    Irrep   Nso     Nmo     Nalpha   Nbeta   Ndocc  Nsocc
   -------------------------------------------------------
     A        117     117       0       0       0       0
   -------------------------------------------------------
    Total     117     117      25      25      25       0
   -------------------------------------------------------

 OEINTS: Overlap, kinetic, potential, dipole, and quadrupole integrals
         stored in file 35.

  ==> Integral Setup <==

  ==> DFJK: Density-Fitted J/K Matrices <==

    J tasked:                  Yes
    K tasked:                  Yes
    wK tasked:                  No
    OpenMP threads:              1
    Integrals threads:           1
    Memory (MB):              1430
    Algorithm:                Core
    Integral Cache:           NONE
    Schwarz Cutoff:          1E-12
    Fitting Condition:       1E-12

   => Auxiliary Basis Set <=

  Basis Set: 
    Number of shells: 222
    Number of basis function: 717
    Number of Cartesian functions: 717
    Spherical Harmonics?: false
    Max angular momentum: 3

  Minimum eigenvalue in the overlap matrix is 4.4641758091E-04.
  Using Symmetric Orthogonalization.
  SCF Guess: Core (One-Electron) Hamiltonian.

  ==> Iterations <==

                           Total Energy        Delta E     RMS |[F,P]|

   @DF-RHF iter   1:  -251.96709197253722   -2.51967e+02   6.52688e-02 
   @DF-RHF iter   2:  -211.47277933759847    4.04943e+01   6.28828e-02 DIIS
   @DF-RHF iter   3:  -256.36868127765308   -4.48959e+01   5.40948e-02 DIIS
   @DF-RHF iter   4:  -295.71109146574827   -3.93424e+01   2.90669e-02 DIIS
   @DF-RHF iter   5:  -301.61345637009458   -5.90236e+00   2.04232e-02 DIIS
   @DF-RHF iter   6:  -305.32541322778741   -3.71196e+00   5.44650e-03 DIIS
   @DF-RHF iter   7:  -305.50214762978260   -1.76734e-01   2.32788e-03 DIIS
   @DF-RHF iter   8:  -305.54457728548437   -4.24297e-02   8.55766e-04 DIIS
   @DF-RHF iter   9:  -305.55198316666110   -7.40588e-03   1.95298e-04 DIIS
   @DF-RHF iter  10:  -305.55238838944649   -4.05223e-04   7.48162e-05 DIIS
   @DF-RHF iter  11:  -305.55244118615065   -5.27967e-05   3.50593e-05 DIIS
   @DF-RHF iter  12:  -305.55245431286517   -1.31267e-05   1.59488e-05 DIIS
   @DF-RHF iter  13:  -305.55245813809631   -3.82523e-06   4.69714e-06 DIIS
   @DF-RHF iter  14:  -305.55245875766047   -6.19564e-07   2.01121e-06 DIIS
   @DF-RHF iter  15:  -305.55245892937592   -1.71715e-07   8.01782e-07 DIIS

  ==> Post-Iterations <==

    Orbital Energies (a.u.)
    -----------------------

    Doubly Occupied:                                                      

       1A    -20.576701     2A    -11.288500     3A    -11.233972  
       4A    -11.233399     5A    -11.226849     6A    -11.226711  
       7A    -11.224455     8A     -1.399367     9A     -1.146555  
      10A     -1.017344    11A     -1.002478    12A     -0.834120  
      13A     -0.825699    14A     -0.735305    15A     -0.680813  
      16A     -0.646514    17A     -0.628676    18A     -0.593278  
      19A     -0.587396    20A     -0.522307    21A     -0.514374  
      22A     -0.490680    23A     -0.463823    24A     -0.336173  
      25A     -0.322305  

    Virtual:                                                              

      26A      0.144885    27A      0.153680    28A      0.231785  
      29A      0.242064    30A      0.293519    31A      0.309013  
      32A      0.333770    33A      0.340574    34A      0.377001  
      35A      0.448536    36A      0.464131    37A      0.480784  
      38A      0.545196    39A      0.552661    40A      0.680721  
      41A      0.732142    42A      0.746131    43A      0.754017  
      44A      0.804935    45A      0.817975    46A      0.827127  
      47A      0.837339    48A      0.840145    49A      0.845266  
      50A      0.886010    51A      0.897580    52A      0.910934  
      53A      0.922396    54A      1.001555    55A      1.034886  
      56A      1.103495    57A      1.115835    58A      1.120097  
      59A      1.129271    60A      1.152907    61A      1.191247  
      62A      1.198980    63A      1.208962    64A      1.240614  
      65A      1.280554    66A      1.290343    67A      1.353750  
      68A      1.391120    69A      1.419583    70A      1.451651  
      71A      1.462093    72A      1.506194    73A      1.618944  
      74A      1.693705    75A      1.713469    76A      1.747152  
      77A      1.769494    78A      1.783845    79A      1.795451  
      80A      1.872801    81A      2.042918    82A      2.089686  
      83A      2.101072    84A      2.156365    85A      2.202546  
      86A      2.267484    87A      2.276038    88A      2.316691  
      89A      2.322612    90A      2.417827    91A      2.457885  
      92A      2.466404    93A      2.473564    94A      2.568016  
      95A      2.617281    96A      2.648126    97A      2.650316  
      98A      2.746056    99A      2.852590   100A      2.909662  
     101A      2.917004   102A      3.025233   103A      3.041984  
     104A      3.089929   105A      3.100984   106A      3.106673  
     107A      3.229762   108A      3.334659   109A      3.513388  
     110A      3.835599   111A      4.324165   112A      4.528363  
     113A      4.546670   114A      4.620155   115A      4.759923  
     116A      4.841660   117A      5.168540  

    Final Occupation by Irrep:
              A 
    DOCC [    25 ]

  Energy converged.

  @DF-RHF Final Energy:  -305.55245892937592

   => Energetics <=

    Nuclear Repulsion Energy =            273.1019261436301804
    One-Electron Energy =                -955.7932172830412583
    Two-Electron Energy =                 377.1388322100357868
    DFT Exchange-Correlation Energy =       0.0000000000000000
    Empirical Dispersion Energy =           0.0000000000000000
    PCM Polarization Energy =               0.0000000000000000
    EFP Energy =                            0.0000000000000000
    Total Energy =                       -305.5524589293752911

    Alert: EFP and PCM quantities not currently incorporated into SCF psivars.

Properties will be evaluated at   0.000000,   0.000000,   0.000000 Bohr
  ==> Properties <==


Properties computed using the SCF density matrix

  Nuclear Dipole Moment: (a.u.)
     X:    -1.9885      Y:     0.9176      Z:     0.0840

  Electronic Dipole Moment: (a.u.)
     X:     1.8210      Y:    -0.2619      Z:    -0.0952

  Dipole Moment: (a.u.)
     X:    -0.1675      Y:     0.6557      Z:    -0.0111     Total:     0.6768

  Dipole Moment: (Debye)
     X:    -0.4258      Y:     1.6666      Z:    -0.0283     Total:     1.7203


  Saving occupied orbitals to File 180.

*** tstop() called on cx1-3-2-1.cx1.hpc.ic.ac.uk at Tue Jul  5 16:26:36 2016
Module time:
	user time   =       6.97 seconds =       0.12 minutes
	system time =       0.06 seconds =       0.00 minutes
	total time  =          7 seconds =       0.12 minutes
Total time:
	user time   =       6.97 seconds =       0.12 minutes
	system time =       0.06 seconds =       0.00 minutes
	total time  =          7 seconds =       0.12 minutes


Properties will be evaluated at   0.000000,   0.000000,   0.000000 Bohr

Properties computed using the  density matrix

  Nuclear Dipole Moment: (a.u.)
     X:    -1.9885      Y:     0.9176      Z:     0.0840

  Electronic Dipole Moment: (a.u.)
     X:     1.8210      Y:    -0.2619      Z:    -0.0952

  Dipole Moment: (a.u.)
     X:    -0.1675      Y:     0.6557      Z:    -0.0111     Total:     0.6768

  Dipole Moment: (Debye)
     X:    -0.4258      Y:     1.6666      Z:    -0.0283     Total:     1.7203

  Quadrupole Moment: (Debye Ang)
    XX:   -37.5886     YY:   -43.0694     ZZ:   -36.6176
    XY:     4.5248     XZ:     0.2198     YZ:     1.3222

  Traceless Quadrupole Moment: (Debye Ang)
    XX:     1.5032     YY:    -3.9775     ZZ:     2.4742
    XY:     4.5248     XZ:     0.2198     YZ:     1.3222

  Mulliken Charges: (a.u.)
   Center  Symbol    Alpha    Beta     Spin     Total
       1     H     0.27010  0.27010  0.00000  0.45981
       2     O     4.38644  4.38644  0.00000 -0.77287
       3     C     2.81649  2.81649  0.00000  0.36703
       4     C     3.12124  3.12124  0.00000 -0.24248
       5     C     3.11978  3.11978  0.00000 -0.23956
       6     H     0.39582  0.39582  0.00000  0.20836
       7     C     3.09654  3.09654  0.00000 -0.19309
       8     H     0.39434  0.39434  0.00000  0.21131
       9     C     3.09668  3.09668  0.00000 -0.19335
      10     H     0.39933  0.39933  0.00000  0.20134
      11     C     3.10567  3.10567  0.00000 -0.21134
      12     H     0.39835  0.39835  0.00000  0.20329
      13     H     0.39923  0.39923  0.00000  0.20154

   Total alpha = 25.00000, Total beta = 25.00000, Total charge = -0.00000


*** PSI4 exiting successfully. Buy a developer a beer!
